JSONAPI/Json/JsonApiFormatter.cs

```diff
@@ -180,6 +180,13 @@ protected void Serialize(object value, Stream writeStream, JsonWriter writer, Js
         {
             writer.WriteStartObject();
 
+            //FIXME: The spec no longer requires that the ID key be "id":

```

Comment:
This in particular I'm unsure of. At the very least if we serialize e.g. `Uuid` as `id`, we should also include it as `uuid`, which it doesn't do yet. Which way to go here?